### DISCUSSION TITLE
Decode the hidden quote 🤐

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1281] [Galaxy Trivia] [answered]
This is a type of a shift cipher in which the message is encrypted by replacing the original letters with “x” number of characters ahead (or before) in the alphabet. Find the original letters to reveal the hidden message. You have finished the puzzle when you have decoded the full message.

```
QCNB AOHM SIO WUH
ECFF NYLLILCMNM,
QCNB YXOWUNCIH SIO
WUH ECFF
NYLLILCMG.
```

**Bonus:** Name the author.

**Comment by tomerf** [comment, marked as answer]
"WITH GUNS YOU CAN KILL TERRORISTS, WITH EDUCATION YOU CAN KILL TERRORISM." was said by Malala Yousafzai
